internal/test/wmcb/deploy/deploy.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: windows-machine-config-bootstrapper
```

Comment:
This should reflect we are running the tests. `wmcb-e2e-test` or something along this lines. Please reflect this elsewhere too.